formulae/nemo.py

```diff
@@ -12,6 +12,7 @@
 from lxml import etree
 from .errors.handlers import e_internal_error, e_not_found_error, e_unknown_collection_error
 import re
+import os
```

Comment:
Are you still using the `os` package here? If not, you should delete the import.